packages/bundle-size/src/reporters/markdownReporter.js

```diff
@@ -0,0 +1,151 @@
+const chalk = require('chalk');
+const fs = require('fs').promises;
+const path = require('path');
+const prettier = require('prettier');
+const { findPackageRoot } = require('workspace-tools');
+
+const { formatBytes } = require('../utils/helpers');
+const sortComparedReport = require('../utils/sortComparedReport');
+
+/** @typedef {import('../utils/compareResultsInReports').ComparedReportEntry} ComparedReportEntry */
+/** @typedef {import('../utils/compareResultsInReports').ComparedReport} ComparedReport */
+
+/**
+ * @param {number} value
+ *
+ * @return {string}
+ */
+function getDirectionSymbol(value) {
+  if (value < 0) {
+    return '<img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/Decrease.svg" />';
+  }
+
+  if (value > 0) {
+    return '<img aria-hidden="true" src="https://microsoft.github.io/sizeAuditor-website/images/icons/IncreaseYellow.svg" />';
+  }
+
+  return '';
+}
+
+/**
+ * @param {import('../utils/calculateDiffByMetric').DiffByMetric} diff
+ *
+ * @return {string}
+ */
+function formatDelta({ delta }) {
+  if (delta === 0) {
+    return '';
+  }
+
+  return `\`${formatBytes(delta)}\` ${getDirectionSymbol(delta)}`;
+}
+
+/**
+ * @param {ComparedReport} report
+ *
+ * @return {{ changedEntries: ComparedReportEntry[], unchangedEntries: ComparedReportEntry[] }}
+ */
+function partitionReport(report) {
```

Comment:
Seems like a more appropriate name, even if it does return unchanged entries too

```suggestion
function getChangedEntries(report) {
```

Reply:
Makes sense to me, I moved out this function and reused it in both reporters, 3687430.